tests/x11/kontact.pm

```diff
@@ -18,6 +18,10 @@ use testapi;
 use version_utils 'is_pre_15';
 
 sub run {
+    # Workaround: Try to fix a race condition between akonadi and kmail
+    # to create the Local mail resources (boo#1105207)
+    x11_start_program('echo -e "[SpecialCollections]\nDefaultResourceId=akonadi_maildir_resource_0" >> ~/.config/specialmailcollectionsrc', valid => 0);
```

Comment:
```suggestion
    x11_start_program('xterm');
    type string '('echo -e "[SpecialCollections]\nDefaultResourceId=akonadi_maildir_resource_0" >> ~/.config/specialmailcollectionsrc')';
```

Reply:
@antlarr The actual problem was this one somehow openQA started to type `ec` then exited and started to type again in krunner. I wonder if it actually makes sense to come up with a followup pr to change all those commands to be typed on an actual xterm window rather than a krunner window... 

![video](https://user-images.githubusercontent.com/229240/56570613-908d9b80-65bb-11e9-97bd-dd90a3fa7ff1.jpg)

Reply:
@ggardet wdyt?

Reply:
open an xterm and type commands inside is probably better.

Reply:
@ggardet we kind of found the reason, see #7321... however I'll add a ticket to address this in the future, but in a more generic way, so that we don't need to apply a workaround on a per-case basis